mrjob/emr.py

```diff
@@ -920,6 +940,8 @@ def cleanup(self, mode=None):
             except Exception as e:
                 log.exception(e)
 
+        # TODO: otherwise, cancel any steps we submitted
```

Comment:
Would it makes sense to add issue-id where it is supposed to be fixed?

Reply:
good point, done